pallets/xyk/src/tests.rs

```diff
@@ -963,7 +963,12 @@ fn discount_sell_fees_should_work() {
 
 		expect_events(vec![
 			Event::PoolCreated(user_1, asset_a, HDX, 10_000, share_token_native, native_pair_account).into(),
-			pallet_asset_registry::Event::Registered(1, bounded_name, AssetType::PoolShare(asset_a, asset_b)).into(),
+			pallet_asset_registry::Event::Registered {
+				asset_id: 1,
+				asset_name: bounded_name,
+				asset_type: AssetType::PoolShare(asset_a, asset_b),
+			}
+			.into(),
```

Comment:
Not part of this PR - this is due to recent changes in warehouse and it was yet updated in Basilisk.

UI needs to be aware of this change.

I wonder it would be better to do it as part of separate PR so it can be easily tracked.

Reply:
It should be done as part of #459

Reply:
I guess we'll wait for #459 before merging this?

Reply:
yes. it will be merged shortly.